docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,85 @@
+
+In addition to being able to target a variety of different device platforms, there was also pressure from the server side to reduce the overall footprint, and more importantly, surface area, of the .NET Framework. By factoring the CoreFX libraries and allowing individual applications to pull in only those parts of CoreFX they require (a so-called "pay-for-play" model), server-based applications built with ASP.NET 5 can minimize their dependencies. This, in turn, reduces the frequency with which patches and updates to the framework will impact these applications, since only changes made to the individual pieces of CoreFX leveraged by the application will impact the application. A smaller deployment size for the application is a side benefit, and one that makes more of a difference if many applications are deployed side-by-side on a given server.
+
+..note The overall size of .NET Core doesn't intend to be smaller than the .NET Framework over time, but since it is pay-for-play, most applications that utilize only parts of CoreFX will have a smaller deployment footprint.
+
+.NET Core and ASP.NET
+^^^^^^^^^^^^^^^^^^^^^
+
+ASP.NET 5 can target either the .NET Framework or .NET Core. In fact, ASP.NET 5 projects can be cross-compiled, targeting both of these frameworks in a single project, and this is how the project templates ship with Visual Studio 2015. For example, the :keyword:`frameworks` section of *project.json* in a new ASP.NET 5 web project will target *dnx451* and *dnxcore50* by default:
+
+.. code-block:: javascript
+
+	"frameworks": {
+		"dnx451": { },
+		"dnxcore50": { }
+	},
+
+*dnx451* represents the .NET Framework, while *dnxcore50* represents .NET Core 5 (5.0). You can use compiler directives (**#if**) to check for symbols that correspond to the two frameworks: :keyword:`DNX451` and :keyword:`DNXCORE50`. If for instance you have code that uses resources that are not available as part of .NET Core, you can surround them in a conditional compilation directive:
+
+.. code-block:: c#
+
+	#if DNX451
+		// utilize resource only available with .NET Framework
+	#endif
+
+The recommendation from the ASP.NET team is to target both frameworks with new applications. If you want to only target .NET Core, remove *dnx451*, or only target .NET Framework, remove *dnxcore50*, from the *frameworks* listed in *project.json*. Note that ASP.NET 4.6 and earlier target and require the .NET Framework, as they always have.
+
+.NET Core and NuGet
```

Comment:
We should eventually add some content here on how to author NuGet packages that run on .NET Core, but take no dependency on DNX. Filed https://github.com/aspnet/Docs/issues/82 to track.